packages/example/src/pages/GridWithLongNodesPage.tsx

```diff
@@ -34,32 +41,56 @@ export const GridWithLongNodesPage = () => {
   );
 };
 
-const FirstRow = () => {
+const FirstRow = forwardRef<SpatialNavigationNodeRef>((_, ref) => {
```

Comment:
I don't think you need a forward ref here, do you? Why don't you put a props named `firstItemRef`?

Reply:
(Actually I'm not even sure of what I'm saying 🙈 )